Makefile

```diff
@@ -53,17 +69,14 @@ install: build
 # ================================= Testing ====================================
 # unit tests (hermetic)
 unit:
-	hack/ci/unit.sh
+	hack/make-rules/unit.sh
 ################################################################################
 # ================================= Cleanup ====================================
-# cleans the cache volume
-clean-cache:
-	$(DOCKER) volume rm -f kind-build-cache >/dev/null
-# cleans the output directory
-clean-output:
-	rm -rf $(OUT_DIR)/
 # standard cleanup target
-clean: clean-output clean-cache
+clean:
+	rm -rf $(OUT_DIR)/
+	# TODO: remove this in the future. We no longer populat vendor
```

Comment:
nit: dedent
this currently prints out as a shell command

Reply:
fixed